services/whatpulse/whatpulse.service.js

```diff
@@ -0,0 +1,127 @@
+import Joi from 'joi'
+import { BaseJsonService, NotFound } from '../index.js'
+
+const schema = Joi.object({
+  Keys: Joi.alternatives(Joi.string(), Joi.number()),
+  Clicks: Joi.alternatives(Joi.string(), Joi.number()),
```

Comment:
I find it interesting that 'Keys' and 'Clicks' in the API response for the`user` are a string,
and for the `team` - they are a number.

compare for example:
https://api.whatpulse.org/team.php?team=reddit&format=json
&
https://api.whatpulse.org/user.php?user=jerone&format=json

Reply:
:see_no_evil: